cli/internal/runsummary/run_summary.go

```diff
@@ -16,7 +25,19 @@ type RunSummary struct {
 	TurboVersion      string             `json:"turboVersion"`
 	GlobalHashSummary *GlobalHashSummary `json:"globalHashSummary"`
 	Packages          []string           `json:"packages"`
-	Tasks             []TaskSummary      `json:"tasks"`
+	Tasks             *[]*TaskSummary    `json:"tasks"`
+}
+
+// Save saves the run summary to a file
+func (summary *RunSummary) Save(singlePackage bool) error {
+	json, err := summary.FormatJSON(singlePackage)
+	if err != nil {
+		return err
+	}
+
+	nonce := "abcdef"
+	filename := fmt.Sprintf("vercel-turbo-run-%s-%s.json", nonce, time.Now().Format(time.RFC3339))
+	return ioutil.WriteFile(filename, json, 0644)
```

Comment:
We need to write to either `<cwd>/.turbo` or change the things we `.gitignore` otherwise `turbo run build && turbo run build` will not play nicely.

Reply:
good point. I've moved this into `.turbo/runs` as #1707 did also. Started a thread in slack as well because it impacts vercel integration